ImperatorToCK3/Data_Files/configurables/government_map.txt

```diff
@@ -7,14 +7,16 @@
 #	ck3 = CK3 government ID
 #	ir = Imperator government ID
 #	irCulture = Imperator culture ID (optional)
-#	has_ck3_dlc = CK3 DLC ID (optional, supported IDs: roads_to_power)
+#	has_ck3_dlc = CK3 DLC ID (optional; supported IDs: roads_to_power)
+#	ck3_title_rank = CK3 title rank tag is converted to (optional; allows b,c,d,k,e)
 #}
-# multiple entries allowed for: ir, irCulture, has_ck3_dlc
+# multiple entries allowed for: ir, irCulture, has_ck3_dlc, ck3_title_rank

```

Comment:
I think writing it like this may be confusing because currently only one `ck3_title_rank` entry is expected, so it'd probably be best to remove it from here and only keep the explanation on line 16.